src/Main.hs

```diff
@@ -34,9 +32,10 @@ opts = info (options <**> helper) $ fullDesc
   <> header "Echidna"
 
 main :: IO ()
-main = do (Options f c cov conf) <- execParser opts
-          cfg          <- maybe (pure defaultConfig) parseConfig conf
-          Campaign r _ <- runReaderT (loadSolTests f (pack <$> c) >>= \(v,w,ts) -> ui v w ts) $
-                            cfg & cConf %~ if cov then \k -> k {knownCoverage = Just mempty} else id
-          if any (/= Passed) $ snd <$> r then exitWith $ ExitFailure 1
-                                         else exitSuccess
+main = do Options f c conf <- execParser opts
+          cfg <- maybe (pure defaultConfig) parseConfig conf
+          res <- flip runReaderT cfg $ do
+            cs       <- contracts f
+            (v,w,ts) <- loadSpecified (pack . (f ++) . (':' :) <$> c) cs >>= prepareForTest
+            view tests <$> ui v w ts (Just $ mkGenDict 0.15 (extractConstants cs) [])
```

Comment:
Should we make that `0.15` a config parameter? I don't see a lot of users tweaking that, but it could be useful for debugging ..

Reply:
Now that we're trying to do releases, I'm gonna try to keep stuff like that out of config until we need to put it in there